Sources/NIOCrashTester/EventLoopCrashTests.swift

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIO
+import Foundation
+
+fileprivate let group = MultiThreadedEventLoopGroup(numberOfThreads: 2)
+
+struct EventLoopCrashTests {
+    let testMultiThreadedELGCrashesOnZeroThreads = CrashTest(
+        regex: "^Precondition failed: numberOfThreads must be positive"
+    ) {
+        try? MultiThreadedEventLoopGroup(numberOfThreads: 0).syncShutdownGracefully()
+    }
+
+    let testWaitCrashesWhenOnEL = CrashTest(
```

Comment:
note to reviewers: these files are the core. To add a new crash test, the only thing you need to do is

```
let testWhatYouAreTesting = CrashTest(regex: "^outputYouExpect") {
    codeThatWillCrash()
}
```

Reply:
the will automatically be collected through the `Mirror` of the test struct. If you add a new test suite, then you'll need to edit `main.swift`.